pkg/middleware/org_redirect_test.go

```diff
@@ -30,7 +30,7 @@ func TestOrgRedirectMiddleware(t *testing.T) {
 		}
 
 		sc.m.Get("/", sc.defaultHandler)
-		sc.fakeReq("GET", "/?orgId=3").exec()
+		sc.fakeReq("GET", "/?orgId=3&kiosk").exec()
 
 		assert.Equal(t, 302, sc.resp.Code)
```

Comment:
you need to check also that the `sc.resp.Header()["Location"]` is the expected (it contains `&kiosk` instead of `&kiosk=`

Reply:
sorry, I can not understand it, can you help me please.

Reply:
I pushed some changes to resolve this.